shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-dialect/shardingsphere-data-pipeline-mysql/src/main/java/org/apache/shardingsphere/data/pipeline/mysql/ingest/client/MySQLClient.java

```diff
@@ -215,7 +218,7 @@ public synchronized AbstractBinlogEvent poll() {
     @SuppressWarnings("unchecked")
     private <T> T waitExpectedResponse(final Class<T> type) {
         try {
-            Object response = responseCallback.get();
+            Object response = responseCallback.get(10, TimeUnit.SECONDS);
```

Comment:
`10` seconds might be too much, could we use the same timeout with connect timeout?